addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15919,7 +15929,7 @@ msgstr ""
 #. label of a setting for the stereoscopic 3D mode of the GUI that is/should be applied
 #: system/settings/settings.xml
 msgctxt "#36500"
-msgid "Stereoscopic 3D mode (current)"
+msgid "Current stereoscopic 3D mode"
```

Comment:
No, rewording still won't suite. Just leave it as is for now.
Regarding 2D/3D: well, from a consumers perspective it probably would because the marketing industry of consumer products is hyping these namings, but it's also not to confuse with 3D in the sense of 3D game engines, that's why I think decision was made back then to use the correct technical wording for it, which is stereoscopic 3D. At some point we could even have a 3D (in the sense of game engine) GUI and a 2D GUI, or a 2D GUI in with some stereoscopic 3D depths. So let's stick technically correct for now.